frodo.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?